sysutils/upt/Portfile

```diff
@@ -47,4 +47,11 @@ post-destroot {
     xinstall -d ${destroot}${docdir}
     xinstall -m 0644 -W ${worksrcpath} README.md LICENSE CHANGELOG \
         ${destroot}${docdir}
+
+    # link man page; must be only one upt version at time
+    set branch      [join [split ${python.version} {}] {.}]
+    set framewpath  ${frameworks_dir}/Python.framework
+    set framewdir   ${framewpath}/Versions/${branch}
+    ln -s ${framewdir}/share/man/man1/${name}.1 \
```

Comment:
Doesn't `${python.prefix}` do the trick here?

Reply:
built in the same way but shorter; thank you!